test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -576,7 +576,7 @@
             (is (nil? second-jetty-server)
                 "Jetty server was unexpectedly attached to the service context")
             (is (thrown?
-                  BindException
+                  IOException
```

Comment:
Suggest adding a bit to the commit message explaining the motivation for the change.  And not sure yet, but this might affect pdb (though possibly just our tests).

Reply:
(For pdb, I suspect it'll be a trivial adjustment, if we need one, but wondered if it might affect any other things.)